pkgs/tools/networking/globalprotect-openconnect/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub,
+  qmake,
+  qtwebsockets, qtwebengine,
+  openconnect } :
+
+stdenv.mkDerivation rec {
+  name = "globalprotect-openconnect-${version}";
+  version = "1.2.6";
+
+  src = fetchFromGitHub {
+    owner = "yuezk";
+    repo = "GlobalProtect-openconnect";
+    fetchSubmodules = true;
+    rev = "c14a6ad1d2b62f8d297bc4cfbcb1dcea4d99112f";
+    sha256 = "1zkc3vk1j31n2zs5ammzv23dah7x163gfrzz222ynbkvsccrhzrk";
+  };
+
+  nativeBuildInputs = [ qmake ];
+
+  buildInputs = [ openconnect
+                  qtwebsockets qtwebengine ];
```

Comment:
```suggestion
  buildInputs = [ openconnect qtwebsockets qtwebengine ];
```

Reply:
You forgot to fix this.

Reply:
It's on the `globalprotect-vpn-formatting` branch as https://github.com/jerith666/nixpkgs/commit/85ee6e0c667bcbcba33ad29648be1b940879b9ca.

Reply:
Can we just put this into this PR?